ports/servo/glutin_app/window.rs

```diff
@@ -561,6 +560,27 @@ impl Window {
         mem::replace(&mut *self.event_queue.borrow_mut(), Vec::new())
     }
 
+    pub fn handle_servo_events(&self, events: Vec<EmbedderMsg>) {
+        for event in events {
+            match event {
+                EmbedderMsg::Status(top_level_browsing_context, message) => self.status(top_level_browsing_context, message),
+                EmbedderMsg::ChangePageTitle(top_level_browsing_context, title) => self.set_page_title(top_level_browsing_context, title),
+                EmbedderMsg::MoveTo(top_level_browsing_context, point) => self.set_position(top_level_browsing_context, point),
+                EmbedderMsg::ResizeTo(top_level_browsing_context, size) => self.set_inner_size(top_level_browsing_context, size),
+                EmbedderMsg::AllowNavigation(top_level_browsing_context, url, response_chan) => self.allow_navigation(top_level_browsing_context, url, response_chan),
```

Comment:
I would be surprised if `./mach test-tidy` doesn't complain about these lines.

Reply:
That's an intermediate commit. All commits compile, but I'm not sure they all pass the tidy test.